kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -218,6 +226,8 @@ def create(  # noqa: PLR0913, PLR0912
             # topological sort order obtained from pipeline.nodes, see group_memory_nodes()
             # implementation
             nodes, dependencies = group_memory_nodes(context.catalog, pipeline)
+        elif group_by_namespace:
+            nodes, dependencies = group_by_namespace(pipeline)
```

Comment:
Should this call the method that @ankatiyar added to the framework?

Reply:
it's calling `Pipeline.grouped_nodes_by_namespace`, I based it on Ankita's prototype

Reply:
Ahh I see now. I guess I didn't expect there still be so much code needed to process the result from `pipeline.grouped_nodes_by_namespace`